application/src/main/java/org/togetherjava/tjbot/commands/tags/TagCommand.java

```diff
@@ -33,19 +35,22 @@ public TagCommand(TagSystem tagSystem) {
 
         // TODO Thing about adding an ephemeral selection menu with pagination support
         // if the user calls this without id or similar
-        getData().addOption(OptionType.STRING, ID_OPTION, "the id of the tag to display", true);
+        getData().addOption(OptionType.STRING, ID_OPTION, "the id of the tag to display", true)
+            .addOption(OptionType.USER, MENTION_USER, "replying to the user ", false);
```

Comment:
the description reads a bit weird from a user perspective. maybe change to _"optionally, the user who you want to reply to"_